### PR TITLE
[#426] Complete Robot Arm UI

### LINKED ISSUE #426: Implement basic UI structure for robotic arm
* **I'm submitting a ...**
  - [ ] bug report
  - [x] feature request

* **What is the current status?**
- The robotic arm does not exist in the desktop app

* **What is the expected behaviour?**
- This should be added as a new feature

* **Please tell us about your environment:**
  - OS with version number: Ubuntu 18.04

* **Would you like to work on it?**
Yes

### PR DESCRIPTION
**Pull request by AakashMallik**
Fixes #426 

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [x] Feature implementation
- [ ] Doc updates

* **What changes have you introduced?**
- Basic robot arm interface created
- Custom dialogue added to change the value of each servo
- Drag and paint option for servo values

* **Does this PR introduce a breaking change?**
No

* **Preview / Steps to verify your work**:

### CODE REVIEW

File: src/screen/Sensors/Sensors.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  Wrapper,
+  HelpText,
+  SecondaryContentWrapper,
+  TitleWrapper,
+  ScrollWrapper,
+  SensorTab,
+  SensorTitle,
+} from './styles';
+
+class Sensors extends Component {
```
Comment (CloudyPadmal) [inline src/screen/Sensors/Sensors.js]: Update the branch

Comment (AakashMallik) [reply src/screen/Sensors/Sensors.js]: @CloudyPadmal Done

Comment (CloudyPadmal) [reply src/screen/Sensors/Sensors.js]: I dunno what do you mean by done? Are you doing everything in same branch? These stuff are related to #431

**Comment by AakashMallik** [comment]
@CloudyPadmal Sir, I am really sorry, this was an accidental merge and can't be undone as I squashed the commit. It should not be a problem when the two PRs get merged, they have literally the same code for the sensor part.

**Comment by CloudyPadmal** [comment]
Now rebase and update the PR

**Comment by AakashMallik** [comment]
> Now rebase and update the PR

Thank you, sir, for understanding... I have rebased the code @CloudyPadmal

**Comment by AakashMallik** [comment]
>     * Make font in black boxes bigger
> 
>     * Add units
> 
>     * `Undefined` values for timeline beyond set points

I have made the changes


i hope this looks fine now.
@CloudyPadmal

File: scripts/robotic_arm.py [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
     def setServo(self, a1, a2, a3, a4):
+        if a1 is None:
+            a1 = self.prev_angle[0]
+        else:
+            self.prev_angle[0] = a1
+
+        if a2 is None:
+            a2 = self.prev_angle[1]
+        else:
+            self.prev_angle[1] = a1
```
Comment (CloudyPadmal) [inline scripts/robotic_arm.py]: Not a1

**Pull request merged**